index.html

```diff
@@ -882,22 +892,28 @@ <h2>
         are defined in [[!DOM]].
       </p>
       <p>
-        <dfn data-cite="!WEBIDL#aborterror"><code>AbortError</code></dfn>,
-        <dfn data-cite="!WEBIDL#notsupportederror">NotSupportedError</dfn>,
-        <dfn data-cite="!WEBIDL#dfn-DOMException">DOMException</dfn>,
+        <code><dfn data-cite="!WEBIDL#aborterror">AbortError</dfn></code>,
+        <code><dfn data-cite="!WEBIDL#notsupportederror">NotSupportedError</dfn></code>,
+        <code><dfn data-cite="!WEBIDL#dfn-DOMException">DOMException</dfn></code>,
         <dfn data-cite="!WEBIDL#SecureContext">Secure context</dfn>,
-        <dfn data-cite="!WEBIDL#notallowederror">NotAllowedError</dfn> and
+        <code><dfn data-cite="!WEBIDL#notallowederror">NotAllowedError</dfn></code> and
         <dfn data-cite=
         "!WEBIDL#dfn-available-only-in-secure-contexts">available only in
         secure context</dfn>
         are defined in [[!WEBIDL]].
       </p>
       <p>
         The terms <dfn data-cite="!PERMISSIONS#query-a-permission">query a permission</dfn>,
-        <dfn data-cite="!PERMISSIONS##request-permission-to-use">request permission</dfn>,
+        <dfn data-cite="!PERMISSIONS#request-permission-to-use">requesting permission to use</dfn>,
```

Comment:
nit: you don't need "!" in front of references anymore. Can remove them globally.